.github/workflows/main.yaml

```diff
@@ -0,0 +1,34 @@
+name: Node.js CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: 21.x
+
+    - name: Setup OS dependencies
+      run: |-
+        sudo apt-get update
+        sudo apt-get install -yy \
+          libnss3 \
+          libatk-bridge2.0-0 \
+          libgtk-3-dev \
+          libasound2 \
+          xvfb
```

Comment:
Whats all this for?

Reply:
For running unit tests in headless mode.